training/src/main/java/wolox/training/models/Client.java

```diff
@@ -20,9 +21,12 @@
     @Column(nullable = false)
     private LocalDate birthdate;
 
-    @OneToMany(mappedBy = "client")
-    private List<Book> books;
-
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_client",
```

Comment:
Unnecessary, given it already infers a JoinTable because it is a ManyToMany relationship. In these cases, this annotation is only useful to assign custom names to the join table and its columns. So remove it :)